ui/src/app/pages/profile/data-user-code-of-conduct.tsx

```diff
@@ -21,70 +59,209 @@ interface Props {
 
 interface State {
   name: string;
-  initialName: string;
-  initialWork: string;
-  initialSanctions: string;
+  initialNameV1: string;
+  initialWorkV1: string;
+  initialSanctionsV1: string;
+  initialMonitoring: string;
+  initialPublic: string;
+  page: DataUserCodeOfConductPage;
   submitting: boolean;
 }
 
-export const DataUseAgreement = withUserProfile()(
+export const DataUserCodeOfConduct = withUserProfile()(
   class extends React.Component<Props, State> {
     constructor(props) {
       super(props);
       this.state = {
         name: '',
-        initialName: '',
-        initialWork: '',
-        initialSanctions: '',
+        initialNameV1: '',
+        initialWorkV1: '',
+        initialSanctionsV1: '',
+        initialMonitoring: '',
+        initialPublic: '',
+        page: DataUserCodeOfConductPage.CONTENT,
         submitting: false
       };
     }
 
-    submitDataUseAgreement(initials) {
+    submitDataUserCodeOfConduct(initials) {
       this.setState({submitting: true});
+      const dataUseAgreementVersion = serverConfigStore.getValue().enableV2DataUserCodeOfConduct ? 3 : 2;
```

Comment:
I'd probably use V3 in the flag name instead of V2. Should hopefully be an easy search/replace. This line illustrates why the current naming could be confusing.

Alternatively: If the intent was to convey "v2" of the DUA implementation, perhaps something like enablePdfDataUser.... would be appropriate.